client/src/components/SideBanner/SideBanner.css

```diff
@@ -0,0 +1,19 @@
+.side-banner {
```

Comment:
Since you're using Material UI you might consider converting styling to [makeStyles](https://material-ui.com/styles/basics/#hook-api), that way you can predefine values for colors, typography, spacing, etc.

Reply:
@camba95 I believe you made these changes to makeStyles as suggested so you can go ahead and remove this file :)

Reply:
dang 😂  Did I not do it? , working on